src/arvcamera.c

```diff
@@ -1823,6 +1823,43 @@ arv_camera_get_gain_auto (ArvCamera *camera, GError **error)
 	return arv_auto_from_string (arv_camera_get_string (camera, "GainAuto", error));
 }
 
+/**
+ * arv_camera_select_gain:
+ * @camera: a #ArvCamera
+ * @gainSelector: gain #ArvSelector selection
+ * @error: a #GError placeholder, %NULL to ignore
+ *
+ * Configures Gain Selector feature.
+ *
+ * Since: 0.8.23
+ **/
+
+void
+arv_camera_select_gain (ArvCamera *camera, const char *gainSelector, GError **error) {
+	arv_camera_set_string (camera, "GainSelector", gainSelector, error);
+
```

Comment:
Opening bracket should on its own line. Closing bracket is missing.

Reply:
thanks, working too fast.